backend/FwLite/FwDataMiniLcmBridge/Api/RichTextMapping.cs

```diff
@@ -133,7 +134,15 @@ public static void WriteToSpan(RichSpan span, ITsTextProps textProps, Func<int?,
             if (IntProps.Contains(propType))
             {
                 MapToDelegateInt(propType).Invoke(span) = value;
+                //these types have values mapped from variation to unit
+                if (propType is FwTextPropType.ktptFontSize or FwTextPropType.ktptOffset or FwTextPropType.ktptLineHeight)
```

Comment:
I would have put the `if (IntProps.Contains)` at the end of the switch. Then you wouldn't have to special case these.